ScienceDirect.js

```diff
@@ -884,23 +884,6 @@ var testCases = [
 			}
 		]
 	},
-	{
-		"type": "web",
-		"url": "http://www.sciencedirect.com/science/handbooks/18745709",
-		"defer": true,
-		"items": "multiple"
-	},
-	{
-		"type": "web",
-		"url": "http://www.sciencedirect.com/science/referenceworks/9780080437484",
-		"items": "multiple"
-	},
-	{
-		"type": "web",
-		"url": "http://www.sciencedirect.com/science/bookseries/00652458",
-		"defer": true,
-		"items": "multiple"
-	},
 	{
 		"type": "web",
 		"url": "http://www.sciencedirect.com/science/article/pii/0584853976801316",
```

Comment:
Switch to `https` for all tests.

Reply:
the site still runs on http, too; also, a number of tests don't run correctly because of https://github.com/zotero/zotero/issues/1502, so can't really update those.

Reply:
It is reachable through http, but you get redirected to https. True, about the problem with updating, thus let us postpone this.